mechanics/RELEASE-SCHEDULE.md

```diff
@@ -12,54 +12,55 @@ The Knative project releases quarterly, on Tuesday of the 4th week of January, A
-| 1.10    | 2023-04-25 | 2023-10-31 | 1.24            | |
-| 1.11    | 2023-07-25 | 2024-01-30 | 1.25            | |
-| 1.12    | 2023-10-24 | 2024-04-30 | 1.26            | |
-| 1.13    | 2024-01-23 | 2024-07-30 | -               | |
-| 1.14    | 2024-04-23 | 2024-10-29 | -               | |
-| 1.15    | 2024-07-23 | 2025-01-28 | -               | |
-| 1.16    | 2024-10-22 | 2025-04-29 | -               | |
+| Release | Date       | EOL        | Min K8s Version | Notes |
+| ------- | ---------- | ---------- | --------------- | ----- |
+| 1.10    | 2023-04-25 | 2023-10-31 | 1.24            |       |
+| 1.11    | 2023-07-25 | 2024-01-30 | 1.25            |       |
+| 1.12    | 2023-10-24 | 2024-04-30 | 1.26            |       |
+| 1.13    | 2024-01-23 | 2024-07-30 | -               |       |
+| 1.14    | 2024-04-23 | 2024-10-29 | -               |       |
+| 1.15    | 2024-07-23 | 2025-01-28 | -               |       |
+| 1.16    | 2024-10-22 | 2025-04-29 | -               |       |
 
 ## Releases supported by community
 
-| Release | Date       | EOL        | Min K8s Version | Notes                    |
-| ------- | ---------- | ---------- | --------------- | ------------------------ |
-| 1.9     | 2023-01-24 | 2023-08-01 | 1.24            | |
-| 1.8     | 2022-10-18 | 2023-05-02 | 1.23            | Change to quarterly release cycle |
+| Release | Date       | EOL        | Min K8s Version | Notes |
+| ------- | ---------- | ---------- | --------------- | ----- |
+| 1.10    | 2023-04-25 | 2023-10-31 | 1.24            |       |
+| 1.9     | 2023-01-24 | 2023-08-01 | 1.24            |       |
 
 ## No longer supported releases
 
-| Release | Date       | EOL        | Min K8s Version | Notes                    |
-| ------- | ---------- | ---------- | --------------- | ------------------------ |
-| 1.7     | 2022-08-23 | 2023-02-21 | 1.22            | |
-| 1.6     | 2022-07-12 | 2023-01-10 | 1.22            | |
-| 1.5     | 2022-05-31 | 2022-11-29 | 1.22            | |
-| 1.4     | 2022-04-19 | 2022-10-11 | 1.22            | |
-| 1.3     | 2022-03-08 | 2022-08-30 | 1.21            | |
-| 1.2     | 2022-01-25 | 2022-07-12 | 1.21            | |
-| 1.1     | 2021-12-14 | 2022-05-31 | 1.20            | |
-| 1.0     | 2021-11-02 | 2022-04-19 | 1.20            | |
-| 0.26    | 2021-09-21 | 2022-03-08 | 1.19            | |
-| 0.25    | 2021-08-10 | 2022-01-25 | 1.19            | |
-| 0.24    | 2021-06-29 | 2021-12-14 | 1.19            | |
-| 0.23    | 2021-05-18 | 2021-11-02 | 1.18            | Removes v1beta1 Eventing APIs |
-| 0.22    | 2021-04-06 | 2021-09-21 | 1.18            | |
-| 0.21    | 2021-02-23 | 2021-08-10 | 1.18            | |
+| Release | Date       | EOL        | Min K8s Version | Notes                                          |
+| ------- | ---------- | ---------- | --------------- | ---------------------------------------------- |
+| 1.8     | 2022-10-18 | 2023-05-02 | 1.23            |                                                |
```

Comment:
nit: technically we still support v1.8 for another couple of days, so should we wait to move it down to the unsupported table until after the EOL date?